spec/operators/bufferTime-spec.ts

```diff
@@ -53,6 +53,9 @@ describe('bufferTime operator', () => {
       const e1 = hot('  ---a---b---c---d---e---f---g-----|   ');
       const subs = '    ^--------------------------------!   ';
       const t = time('  ----------|                          ');
+      //                       ----------|
+      //                               ----------|
+      //                                       ----------|
```

Comment:
What are these comments for?

Reply:
They were just to help me visualize/estimate where values should be. A sanity test for the marbles in that test.